cypress/integration/eholdings/notes.spec.js

```diff
@@ -1,24 +1,26 @@
+import { cy } from 'date-fns/locale';
+import { Accordion, Button, including, Link, Page, Pane } from '../../../interactors';
+
 describe('ui-eholdings: Notes', () => {
   before('logs in and navigates to eHoldings', () => {
+    cy.visit('/');
     cy.login('diku_admin', 'admin');
-    cy.visit('/eholdings');
+    cy.do(Link('eHoldings').click());
   });
 
   describe('creating a Note', () => {
     before('searching and opening provider', () => {
       cy.search('EBSCO');
-      cy.get('#search-results-content li:first-child').click();
+      cy.do(Pane('Providers').find(Link(including('EBSCO'))).click());
```

Comment:
Do we need to set this in a `Pane` here or is it enough to just hit the link directly?

Reply:
I think it's the same as for the Accordion comment, but less important to have a Pane call, because that link is more uniq and have some context.